code/__HELPERS/files.dm

```diff
@@ -83,16 +83,13 @@
 				warning("Skipping map [potential] due to  it being too hot outside. Ideal temp is below 40F, found [temperature].")
 				binary = null
 				continue
-//Lamprey currently has a tendency to crash for no real reason, uncomment this when it stops being broken
-/*
 		if(potential == "Lamprey Station/") //Available if the station is wrecked enough
-			var/crew_score = score["crewscore"] //So that we can use this in the chat
-			if(crew_score >= -20000)
+			var/crew_score = score["crewscore"] //So that we can use this in the admin messaging
+			if(crew_score > -20000)
 				message_admins("Skipping map [potential], stationrequires lower than -20000 score (is [score]).")
```

Comment:
you used [score] instead of [crew_score] or [score["crew_score"]]

Reply:
Good catch